tfhe/src/core_crypto/commons/numeric/unsigned.rs

```diff
@@ -40,6 +40,9 @@ pub trait UnsignedInteger:
 {
     /// The signed type of the same precision.
     type Signed: SignedInteger<Unsigned = Self> + CastFrom<Self>;
+    /// Return the leading zeros of the value.
+    #[must_use]
+    fn leading_zeros(self) -> u32;
```

Comment:
Could be named `leading_zeros_count` or `number_leading_zeros`

Reply:
no that's the rust APIs just being forwarded here

Reply:
Ok